manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Runtime
+PackageVersion: 5.5.2.0
+PackageLocale: en-US
+Publisher: Newtek, Inc.
+PublisherUrl: https://ndi.tv/
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: NDI 5 Runtime
+PackageUrl: https://ndi.tv/
+License: MIT
```

Comment:
Can you provide a link to the license? I can't seem to find it on the PackageUrl https://ndi.tv/. If no info about the Licensing can be found, this field should be set to `Proprietary`.